atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/monitoring/TimestampTrackerTest.java

```diff
@@ -119,7 +119,7 @@ public void callsSupplierAgainAfterTimeElapses() {
 
     @Test
     public void doesNotCallSupplierUnlessGaugeIsQueried() {
-        when(mockClock.getTick()).thenReturn(0L, CACHE_INTERVAL_NANOS * 50000);
+        when(mockClock.getTick()).thenReturn(0L, 0L, CACHE_INTERVAL_NANOS * 50000);
```

Comment:
See https://github.com/dropwizard/metrics/compare/v4.1.2...v4.2.12#diff-a4cfca9d44fb7b3861d760c36f063fa151abb2e1ca1055445febd2d79519d5c6. Clock#getTick is now called on CachedGauge init.